extensions/amp-video/0.1/amp-video.js

```diff
@@ -207,7 +208,8 @@ class AmpVideo extends AMP.BaseElement {
     this.propagateAttributes(ATTRS_TO_PROPAGATE_ON_LAYOUT, this.video_,
         /* opt_removeMissingAttrs */ true);
 
-    this.getRealChildNodes().forEach(child => {
+    const children = scopedQuerySelectorAll(this.element, 'source track');
```

Comment:
This is looking for `<track>`s inside a `<source>` inside the video. Did you mean `source,track`?